app/connectors/HmrcTierConnector.scala

```diff
@@ -90,8 +90,8 @@ class HmrcTierConnector @Inject()(client: HttpClient, configuration: Configurati
     val optMapped = Map(HeaderTags.ETAG -> etagFromSession, HeaderTags.X_TXID -> xtxidFromSession)
 
     Logger.info(
```

Comment:
plz fix the Logger deprecated warnings by either extends Logging or Logger(this.getClass)